index-reduce-v3.js

```diff
@@ -0,0 +1,58 @@
+const students = [
+	{
+		name: "Peyton",
+		grade: 47,
+	},
+	{
+		name: "Emily",
+		grade: 56,
+	},
+	{
+		name: "Claus",
+		grade: 35,
+	},
+	{
+		name: "Nelly",
+		grade: 72,
+	},
+	{
+		name: "Brandon",
+		grade: 80,
+	},
+	{
+		name: "Trent",
+		grade: 79,
+	},
+	{
+		name: "Chris",
+		grade: 61,
+	},
+	{
+		name: "James",
+		grade: 60,
+	},
+	{
+		name: "Alex",
+		grade: 86,
+	},
+	{
+		name: "Khloe",
+		grade: 97,
+	},
+];
+
+const passingScore = 60;
+const highestScore = (scoreA, scoreB) => Math.max(scoreA, scoreB);
+
+const getTheHighestScore = students
+	.filter((student) => student.grade >= passingScore)
+	.map((student) => student.grade)
+	.reduce(highestScore);
+
+const getNameOfHighestScoreStudent = students
```

Comment:
You don't need this part cus I only asked for the score.

But let's say you wanted to have the name and highest score. Then you would have to write more logic to your callback function for highestScore instead of just letting it call Math.Max.

Reply:
Got it